### PR TITLE
Styling the [WhyPaused] UI

### PR DESCRIPTION
**Pull request by krishnagopinath**
Associated Issue: #1604 

Just minor CSS changes such as 
* background
* border
* font-size

### CODE REVIEW

**Comment by krishnagopinath** [comment]
I couldn't check this layout in vertical mode because, try as I could, I couldn't make the layout to switch to that mode. Wasn't it supposed to "flip" when the width was at 700px?

**Comment by jasonLaster** [comment]
@krishnagopinath you can turn on vertical mode in local.json. 

Checkout our local development doc for more info.

File: src/components/WhyPaused.css [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
   padding: 10px 10px 10px 20px;
   white-space: normal;
+  opacity: 0.9;
+  font-size: 12px;
+  text-align: center;
+  font-weight: bold;
+}
+
+.theme-dark .why-paused {
+  color: white !important;
```
Comment (jasonLaster) [inline src/components/WhyPaused.css]: Haha can we add more specificity to avoid imports to calls?

**Comment by irfanhudda** [comment]
@krishnagopinath It probably means that you need to avoid using "!important" whenever possible.  It is not considered as good practice. 

Most of the times you can avoid them by using better(more specific) CSS selectors.

**Comment by krishnagopinath** [comment]
@jasonLaster check the latest changes. Tell me if I need to change anything more

**Comment by jasonLaster** [comment]
Here are some screens

screen shot 2017-01-09 at 10 00 25 am
screen shot 2017-01-09 at 9 59 39 am

**Pull request merged**